tutorials/3d/3d_text.rst

```diff
@@ -0,0 +1,38 @@
+.. _doc_3d_text:
+
+Introduction
+============
+
+In a project there may be times when text needs to be created as
+part of a 3D scene and not just in the HUD. Godot provides two
+methods to do this. The Label3D node and the text mesh for a
+MeshInstance3D node.
+
+This page does **not** cover how to display a GUI scene in a 3D
+environment. For information on how to do that see `this <https://github.com/godotengine/godot-demo-projects/tree/master/viewport/2d_in_3d>`__
+demo project.
+
+Label3D
+-------
+
+.. image:: img/label_3d.png
+
+Label3D behaves like a label node but in a 3D space. Unlike label
+node this can not inherit properties of a GUI theme. However its
+look remains customizable and uses the same font subresource
+control nodes use.
+
+Label3D has minimal interaction with a 3D environment, it can be lit
+up and shaded by light sources if the shaded flag is enabled, but it
+will not cast a shadow, even with cast shadow turned on under the nodes
+GeometryInstance3D settings.
+
```

Comment:
I guess it worth mentioning that it's quad mesh (one quad per glyph) with transparent textures and therefore have the same transparency sorting limitations as Sprite3D (with the link to https://docs.godotengine.org/en/stable/tutorials/3d/3d_rendering_limitations.html?highlight=transparancy#transparency-sorting).